start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,151 @@
+function NewEvaluationPage(options) {
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    <section class="form-candidate">
+    <div class="container">
+       <div class="row">
+            <div class="col-md-12">
+                <form action="" class="user-input" id='form-new' method='post'>
+    ${Inputs(
```

Comment:
why not indent this as well so they are on the same level as their siblings?

Reply:
Solved.